README.md

```diff
@@ -24,11 +24,34 @@ The EVE project welcomes all contributions!
 4. Thanks you in advance :thumbsup: !!
 
 ## Configuration
-There is a [`config.py`](/eve/config.py) that can be customised to support different use cases.
-
-Override the relevant variables to fit your use case.
+In order to use all supported commands, you'd need to provide your personal preferences/credentials.
+For convenience, create ~/.eve_config file with the following content, examples in the comments
+```
+# Weather API
+export WX_API_KEY=you_personal_open_weather_api_key # e5209303a83828u2eako7c849302d2j2
+export WX_LOCATION=your_preferred_default_location # London
+export WX_METRIC_TEMP=your_preferred_temperature_units # celsius ( <celsius|fahrenheit|kelvin> )
+export WX_METRIC_WIND=your_preferred_wind_units # km_hour ( <miles_hour|km_hour|knots> )
+
+#Github API
+export GITHUB_USER=your_github_username # johndoe
+export GITHUB_PASS=your_github_password # supersecretpassword
+
+# Tidy
+export TIDY_ROOT="</path/to/root>" # /User/johndoe/Downloads
+
+#Crypto
+export COIN_MARKET_CAP_API_KEY=your_personal_coinmarket_cap_api_key # 5i3qe43d-2de2-2eid-63b3-9403920395d2
+export DEFAULT_COIN=your_preferred_default_crypto_currency_symbol # BTC
+export DESTINATION_CURRENCY=your_preferred_default_currency_to_convert_to # EUR
+```
 
-**DO NOT COMMIT ANY SENSITIVE INFORMATION!**
+export env variables in this file on the startup of the shell.
+Depending on what shell you use, add this line to `~/.zshrc`; `~/.bash_profile`, `etc`
+```
+source ~/.eve_config
+```
+restart your shell
```

Comment:
This is cool and does make a lot more sense but I don't want people to get confused by watching the videos then seeing this. Can we keep the explanation about setting them in [config.py]?

Do you think it is possible that this can be an optional thing? So you can either suppler environment variables or use values in [config.py]? 

It should be a simple change, just adding defaults to all the values like , `os.environ.get("GITHUB_USER", "<github-username-here>")`

Reply:
Yes, sure, I'll add it to the README

Reply:
I've updated my PR, could you please check and tell if I understood you correctly with this?